### PR TITLE
Adds support for clustered websocket messages for products

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
This PR updates the `HooksForPlugin` function to return product registered hooks as well, and allows for clustered websocket messages to be processed.

#### Release Note
```release-note
* Adds support for product websocket messages on high availability instances.
```

### CODE REVIEW

File: app/cluster_handlers.go [hunk 1]
```diff
@@ -37,6 +37,10 @@ func (s *Server) clusterPluginEventHandler(msg *model.ClusterMessage) {
 		return
 	}
 	pluginID := msg.Props["PluginID"]
+	// if plugin is empty, the message cloud come from a product
```
Comment (isacikgoz) [inline app/cluster_handlers.go:40]:
```suggestion
	// if the plugin key is empty, the message might be coming from a product.
```

**Review by isacikgoz — APPROVED** [review]
LGTM 👍  Small nit, for the sake of 10 lines.

**Review by jespino — APPROVED** [review]
LGTM

**Comment by amyblais** [comment]
@mgdelacroix Is this for v7.6?

**Pull request merged**